addons/controllers/clusterbootstrap_controller_test.go

```diff
@@ -170,26 +170,6 @@ var _ = Describe("ClusterBootstrap Reconciler", func() {
 				Expect(hasPackageInstalls(ctx, k8sClient, cluster, constants.TKGSystemNS,
 					clusterBootstrap.Spec.AdditionalPackages, logr.Logger{})).To(BeTrue())
 
-				By("packageinstalls for additional packages should all have the correct owner reference set")
-				var ownerIsCluster bool
-				pkgInstall := &kapppkgiv1alpha1.PackageInstall{}
-				for _, pkg := range clusterBootstrap.Spec.AdditionalPackages {
-					ownerIsCluster = false
-					pkgInstallName := util.GeneratePackageInstallName(cluster.Name, pkg.RefName)
-					err := k8sClient.Get(ctx, client.ObjectKey{Name: pkgInstallName, Namespace: constants.TKGSystemNS}, pkgInstall)
-					if err == nil {
-						for _, ownerRef := range pkgInstall.OwnerReferences {
-							if ownerRef.UID == cluster.UID {
-								ownerIsCluster = true
-								break
-							}
-						}
-						if !ownerIsCluster {
-							break
-						}
-					}
-				}
-				Expect(ownerIsCluster).To(BeTrue())
```

Comment:
Could we add a test to verify that ownerRefs is empty for all packages except kapp?

Reply:
done.